LoopKit/Persistence/Model.xcdatamodeld/Model.xcdatamodel/contents

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="14492.1" systemVersion="18G87" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="18154" systemVersion="20G80" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
     <entity name="CachedCarbObject" representedClassName=".CachedCarbObject" syncable="YES">
-        <attribute name="absorptionTime" optional="YES" attributeType="Double" usesScalarValueType="NO" syncable="YES"/>
-        <attribute name="createdByCurrentApp" attributeType="Boolean" usesScalarValueType="YES" syncable="YES"/>
-        <attribute name="externalID" optional="YES" attributeType="String" syncable="YES"/>
-        <attribute name="foodType" optional="YES" attributeType="String" syncable="YES"/>
-        <attribute name="grams" attributeType="Double" defaultValueString="0.0" usesScalarValueType="YES" syncable="YES"/>
-        <attribute name="startDate" attributeType="Date" usesScalarValueType="NO" syncable="YES"/>
-        <attribute name="syncIdentifier" optional="YES" attributeType="String" syncable="YES"/>
-        <attribute name="syncVersion" attributeType="Integer 32" defaultValueString="1" usesScalarValueType="YES" syncable="YES"/>
-        <attribute name="uploadState" attributeType="Integer 32" defaultValueString="0" usesScalarValueType="NO" syncable="YES"/>
-        <attribute name="uuid" optional="YES" attributeType="UUID" usesScalarValueType="NO" syncable="YES"/>
+        <attribute name="absorptionTime" optional="YES" attributeType="Double" usesScalarValueType="NO"/>
```

Comment:
There is a pretty significant data migration happening in the DIY sync. Can this go in after that lands?

Reply:
Yep yep!  I'll hold off.  (I also still have some extant changes I'm working on in `Loop` anyway...)